src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerNoticeOfChange.java

```diff
@@ -102,6 +104,36 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
                 .done();
     }
 
+    public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
+        final CaseDetails<CaseData, State> details,
+        final CaseDetails<CaseData, State> detailsBefore
+    ) {
+        CaseData data = details.getData();
+
+        if (NO.equals(data.getNoticeOfChange().getAreTheyRepresented())
+            || !YES.equals(data.getNoticeOfChange().getAreTheyDigital())) {
+            return AboutToStartOrSubmitResponse.<CaseData, State>builder()
+                .data(data)
+                .build();
+        }
+
+        final boolean isApplicant1 = data.getNoticeOfChange().getWhichApplicant() == APPLICANT_1;
+        final Applicant applicant = isApplicant1 ? data.getApplicant1() : data.getApplicant2();
+        String email = applicant.getSolicitor().getEmail();
+        String orgId = applicant.getSolicitor().getOrganisationPolicy().getOrganisation().getOrganisationId();
+
+        List<String> errors = solicitorValidationService.validateEmailBelongsToOrgUser(email, details.getId(), orgId);
+
+        AboutToStartOrSubmitResponse.AboutToStartOrSubmitResponseBuilder<CaseData, State> responseBuilder =
+            AboutToStartOrSubmitResponse.<CaseData, State>builder()
+                .data(data);
+
+        if (!errors.isEmpty()) {
```

Comment:
can you not just put the empty list into .errors(errors)?

Reply:
Had it in my head that an empty list was a problem. It's not you're right - resolved